libs/sdk-ui-tests-e2e/scripts/run_boiler_app.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# (C) 2021-2023 GoodData Corporation
+
+log() {
+  local now;
+  now=$(date '+%Y-%m-%d %H:%M:%S')
+  echo "$now $*" 1>&2
+}
+
+health_check() {
+    for ((i = 1; i <= 50; i++)); do
+        log "Check Boiler app is up, try $i"
+        if curl -k $BOILER_APP_HOST ; then
+            return 0
+        else
+            sleep 3
+        fi
+    done
+    return 1
+}
+
+apt-get update && apt-get install -y curl && apt-get install -y jq && apt-get install -y sed && apt-get clean
+
+npx --yes @gooddata/$BOILER_APP_VERSION init $BOILER_APP_NAME --language $BOILER_APP_LANGUAGE
+cd $BOILER_APP_NAME
+
+tmp=$(mktemp)
+jq --arg host "${HOST}" --arg ws "${TEST_WORKSPACE_ID}" --arg backend "${sdk_backend}" '.gooddata.hostname = $host | .gooddata.workspaceId = $ws | .gooddata.backend = $backend' package.json > $tmp && mv $tmp package.json
```

Comment:
There are 2 things in this line
 - Please use line break `\` so that it's easier to see all the params, i.e https://github.com/gooddata/gooddata-ui-sdk/blob/master/common/scripts/ci/run_cypress_integrated_tests.sh#L58-L60
 - there is race condition with bash, when creating file then edit/move it. so use another line for `mv $tmp package.json`